.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -303,6 +306,25 @@ jobs:
         done
         gsutil cp -r gs://definitions.stackrox.io/v4/vulnerability-bundles/dev/* gs://definitions.stackrox.io/v4/vulnerability-bundles/1.0.0/
 
+    - name: Copy v1 to pre-versioned bundles
+      run: |
+        # using v1 bundle for released versions listed in RELEASE_VERSION
+        single=gs://definitions.stackrox.io/v4/vulnerability-bundles/v1/vulns.json.zst
+        multi=gs://definitions.stackrox.io/v4/vulnerability-bundles/v1/vulnerabilities.zip
+        # Parse all supported pre-4.6 releases and copy the versioned bundle v1 to ensure
+        # these releases get updates.
+        grep -E "^[45]\.[0-9]+\.[0-9]+$" scanner/updater/version/RELEASE_VERSION | while read -r release; do
+          case "$release" in
+          4.4*)
+            gsutil cp "$single" "gs://definitions.stackrox.io/v4/vulnerability-bundles/$release/"
+            ;;
+          *)
+            gsutil cp "$single" "gs://definitions.stackrox.io/v4/vulnerability-bundles/$release/"
```

Comment:
> @jvdm do you think this is necessary? I think in 4.5, we made it configurable, but, in reality, we just use the multi-bundle.

We are being extra careful because users can still disable the multi-bundle env var/feature, but they won't get updates. I agree it's probably fine to move forward without it. Maybe we can look into deprecating the env var?

Reply:
I can make a ticket to deprecate/remove the env var, as I don't see a purpose for it anymore. Based on this, seems like this is ok, and I can resolve the comment